LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
Is this file deleted by some sort of force pull? We'll still need it in the repo